tests/arbitrageTest.py

```diff
@@ -0,0 +1,50 @@
+import json
+import io
+import unittest
+import unittest.mock
+from src import arbitrage
+
+
+class QueryTest(unittest.TestCase):
+    def setUp(self):
+        self.dates = ["01/01/1990", "01/02/1990"]
+        self.spot = 10
+        self.option_prices = {"01/01/1990" : (15, 15), "01/02/1990" : (14, 15)}
+        self.stock = arbitrage.StockData("AAPL", self.dates, self.spot, self.option_prices)
+        self.data = {"AAPL" : self.stock}
+        self.names = ["AAPL"]
+        self.api = arbitrage.FakeAPI(self.data)
+        
+    def test_check(self):
+        q = arbitrage.Query(stock_names=self.names, api = self.api)
+        self.assertTrue(q.check(self.dates, self.option_prices, self.spot))
+
+    @unittest.mock.patch('sys.stdout', new_callable=io.StringIO)
```

Comment:
Maybe better to have find_opportunities return a list and have that list be printed later.

Reply:
fixed